charts/jenkins/values.yaml

```diff
@@ -311,7 +311,7 @@ controller:
       # jcasc changes will cause a reboot and will only be applied at the subsequent start-up.  Auto-reload uses the
       # http://<jenkins_url>/reload-configuration-as-code endpoint to reapply config when changes to the configScripts are detected.
       enabled: true
-      image: kiwigrid/k8s-sidecar:0.1.275
+      image: kiwigrid/k8s-sidecar:1.12.0
```

Comment:
changelog and chart version need bumping

Reply:
done